src/app/pages/client/inbox/Invites.tsx

```diff
@@ -102,6 +108,12 @@ function InviteCard({ room, userId, direct, compact, onNavigate }: InviteCardPro
         <Box grow="Yes">
           <Text size="T200" priority="300" truncate>
             Invited by <b>{senderName}</b>
+            {serverName && (
+              <>
+                {' on '}
+                <b>{serverName}</b>
+              </>
+            )}
```

Comment:
I think displaying only `serverName` wont be sufficient here as we only display `display_name` and the username part could be anything. maybe we should show the entire userId as "Invited by __name__ (@ name:server.name)"